packages/docs/components/atoms/Action/js-api.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Action JS API
+---
+
+# JS API
+
+The `Action` component extends the [`Base` class](https://js-toolkit.studiometa.dev/api/configuration.html). It inherits it respective API, so make sure to take a look at it.
```

Comment:
**chore:** This can be removed

All components extend the `Base` class, I don't think we need to repeat this for every component. Is it interesting when a component extends another one that is not the `Base` class.